src/jazkarta/easyformplugin/jazshop/interfaces.py

```diff
@@ -73,3 +78,19 @@ class ILikert(IField):
 
 class ILikertWidget(z3c.form.interfaces.IWidget):
     """Likert widget."""
+
+
+class IJazShopArbitraryPriceStringField(IField):
+    """Arbitrary price field (suitable for donations)"""
+    available_products = schema.Tuple(
```

Comment:
I dont understand why what is supposed to be a simple donation text field needs to have this available_products here.
and its a required field??

I see that our migrated form contains one entry for this required field
[$1.00 - Donation] 

and the form works, but I dont know why this is needed